src/frontend-pwa/src/views/Report/Report.tsx

```diff
@@ -1,12 +1,152 @@
+    } else if (!isPhoneNumberValid) {
+      setErrorMessage('Tip: Phone number must be between 10 and 16 characters long, if provided');
+    } else {
+      setErrorMessage('');
+    }
+
+    return isValid;
+  }, [eventType, details, phoneNumber]);
+
+  const handleEventTypeChange = (e: { target: { value: React.SetStateAction<string> } }) => {
+    setEventType(e.target.value);
+  };
+
+  const handleDetailsChange = (e: { target: { value: React.SetStateAction<string> } }) => {
+    setDetails(e.target.value);
+  };
+
+  const handlePhoneNumberChange = (e: { target: { value: React.SetStateAction<string> } }) => {
+    setPhoneNumber(e.target.value);
+  };
+
+  const handleSubmit = async (e: { preventDefault: () => void }) => {
+    e.preventDefault();
+
+    const currentTime = new Date();
+
+    const formData = {
+      latitude,
+      longitude,
+      eventType,
+      details,
+      phoneNumber,
+      time: currentTime,
+    };
+
+    try {
+      await axios.post(`${constants.BACKEND_URL}/api/report`, formData);
+      setErrorMessage('');
+      setEventType('');
+      setDetails('');
+      setPhoneNumber('');
+    } catch (error) {
+      throw Error('Error: Form was not submitted');
+    }
+  };
+
+  useEffect(() => {
+    setIsFormValid(checkFormValidity());
+  }, [checkFormValidity]);
+
   return (
-    <div>
-      <h2>
-        Report an Event
-      </h2>
-      <TextField />
-    </div>
+    <form onSubmit={handleSubmit}>
+      <StyledReportOuterDiv>
+        <StyledReportContainer>
+          <h2>Report An Event</h2>
+          <Section>
+            <div>Event Type:</div>
+            <StyledSelect
+              id="eventType"
+              aria-label="event select"
+              value={eventType}
+              onChange={handleEventTypeChange}
+              required
+            >
+              <option value="">Select an event type</option>
+              {eventOptions.map((event: string, index: number) => (
+                <option value={event} key={index}>
+                  {event}
+                </option>
+              ))}
+            </StyledSelect>
+          </Section>
+          <Section>
+            <div>Event Details:</div>
+            <StyledTextArea
+              id="details"
+              aria-label="detail field"
+              value={details}
+              onChange={handleDetailsChange}
+              rows={7}
+              cols={25}
+              required
+            />
+          </Section>
+          <Section>
+            <div>Phone Number (optional): </div>
+            <StyledInput
+              type="text"
+              aria-label="phone field"
+              id="phoneNumber"
+              value={phoneNumber}
+              onChange={handlePhoneNumberChange}
+            />
+          </Section>
+          {errorMessage && <StyledP>{errorMessage}</StyledP>}
+          <Section>
+            <Button text="Submit" variant="primary" size="md" disabled={!isFormValid} />
```

Comment:
This concerns the button component itself, but a good point Angela had made was that the button should have 
`cursor: not-allowed;` added to the `:disabled` pseudo element

Reply:
This goes back to regex'ing the phone number, but I can submit an invalid phone number since the requirement is its length rather than quality.